rbac/management/utils.py

```diff
@@ -213,3 +216,19 @@ def roles_for_cross_account_principal(principal):
         )
         role_names_list = list(role_names)
     return Role.objects.filter(name__in=role_names_list)
+
+
+def create_object_in_tenant(schema_name, associate_tenant, model, **kwargs):
+    """Create object based on model in tenant schema."""
+    tenant = Tenant.objects.get(schema_name=schema_name)
+    with tenant_context(tenant):
+        model_object, created = model.objects.get_or_create(**kwargs)
```

Comment:
After the reconciliation script is run, we'll want this to include the tenant in the get_or_create.